tailwind.config.mjs

```diff
@@ -36,12 +36,12 @@ const config = {
         "light-transparent": "var(--ink-background-light-transparent)",
         "light-invisible": "var(--ink-background-light-invisible)",
       },
-      primary: "var(--ink-primary)",
-      "primary-hover": "var(--ink-primary-hover)",
-      "primary-pressed": "var(--ink-primary-pressed)",
-      secondary: "var(--ink-secondary)",
-      "secondary-hover": "var(--ink-secondary-hover)",
-      "secondary-pressed": "var(--ink-secondary-pressed)",
+      primary: "var(--ink-button-primary)",
+      "primary-hover": "var(--ink-button-primary-hover)",
+      "primary-pressed": "var(--ink-button-primary-pressed)",
+      secondary: "var(--ink-button-secondary)",
+      "secondary-hover": "var(--ink-button-secondary-hover)",
+      "secondary-pressed": "var(--ink-button-secondary-pressed)",
```

Comment:
I think the tailwind variables should also include the prefix, like background, text and status